src/TestCog.py

```diff
@@ -0,0 +1,17 @@
+from discord.ext import commands

```

Comment:
I'm guessing this was just a practice file that you don't need anymore, so remove it.